### PR TITLE
fix: TT-144 Responsive all tables

### PR DESCRIPTION
**Pull request by jr-98**
All tables are now responsive

### CODE REVIEW

**Review by juanultimate — APPROVED** [review]

File: src/app/modules/time-entries/pages/time-entries.component.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <th class="col">Customer</th>
         <th class="col">Project</th>
         <th class="col">Activity</th>
-        <th class="col"></th>
+        <th class="col">Options</th>
```
Comment (juanultimate) [inline src/app/modules/time-entries/pages/time-entries.component.html:28]: This change may broke unit tests

Comment (jr-98) [reply src/app/modules/time-entries/pages/time-entries.component.html:28]: ok, I will reverted this change

Comment (jr-98) [reply src/app/modules/time-entries/pages/time-entries.component.html:28]: OK. I will reverted this change

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.scss [hunk 2]
```diff
@@ -26,3 +26,8 @@
 .hidden-col{
   display: none;
 }
+.table.dataTable th,
```
Comment (scastillo-jp) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.scss:29]: @jr-98 is this style working? dataTable is not a class

Comment (jr-98) [reply src/app/modules/reports/components/time-entries-table/time-entries-table.component.scss:29]: I took as an example a table that had a similar problem and the solution was that and it worked? What should I change?

Comment (jr-98) [reply src/app/modules/reports/components/time-entries-table/time-entries-table.component.scss:29]: @scastillo-jp  The class does work

**Review by josepato87 — APPROVED** [review]

**Review by scastillo-jp — APPROVED** [review]

**Comment by scastillo-jp** [comment]
@jr-98 
I found this problem when the data is loading
Screen Shot 2021-02-05 at 11 12 25 AM

After loading the data, the table does not have the correct margin: 
Screen Shot 2021-02-05 at 11 12 34 AM

**Comment by scastillo-jp** [comment]
Uploading Screen Recording 2021-02-18 at 3.56.46 PM.mov…

**Review by scastillo-jp — APPROVED** [review]

**Pull request merged**